lib/rubocop/cop/rails/transaction_exit_statement.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for the use of exit statements (namely `return`,
+      # `break` and `throw`) in transactions.
```

Comment:
Could you add why these method are not desirable and the alternative `raise` (rollback) or `next` (commit)?